docker/Dockerfile

```diff
@@ -4,14 +4,14 @@
 # NOTE: at the moment we use alpine3.11 instead of alpine latest
 # due to https://github.com/gliderlabs/docker-alpine/issues/539
 # which broke alpine dns lookup methods starting at alpine 3.12.
```

Comment:
split the dockerfile changes to another PR, hopefully we won't need it and we can move straight to @roekatz's changes from this PR: #254